lib/constructs/provisioning-sfn-workflow.ts

```diff
@@ -89,7 +89,7 @@ export class ProvisioningWorkflow extends Construct implements IProvisioningWork
       vpc: props.vpc,
       tracing: lambda.Tracing.ACTIVE,
     });
-    props.idp?.addClient(new IdentityProviderLambdaClient("CspWritePortfolioClient", cspWritePortfolioFn), [
+    props.idp?.addClient(new IdentityProviderLambdaClient("CspWriteToPortfolioClient", cspWritePortfolioFn), [
```

Comment:
I noticed besides the change in the Idp Client name, this appears to have the same changes as https://github.com/dod-ccpo/atat-web-api/pull/1481. Are both PRs needed?

Reply:
we can ignore 1481. That's something I can put a hold on for now.